docs/other/tips/git.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Git
+---
+Tips and Tricks for Git
+
+### Git Submodules
+
+```bash
+// 1. Add submodule to a new repo
+git submodule add https://github.com/chaconinc/DbConnector
+
+// 2. Updating repo with new submodule
+git pull
+git submodule update --init --recursive
+
+// 2. Or Cloning repo with new submodule
+git clone --recurse-submodules https://github.com/chaconinc/MainProject
+
+// 3. Pulling repo with submodules
+git pull --recurse-submodules
+```
+
+by [git](https://git-scm.com/book/en/v2/Git-Tools-Submodules)
+
+### Delete branch locally and remotely
+
+```bash
+// 1. locally
+git branch -d localBranchName
+
+// 2. remote
+git push origin :remoteBranchName
+```
+
+by [FreeCodeCamp](https://forum.freecodecamp.org/t/how-to-delete-a-git-branch-both-locally-and-remotely/13211)
+
+### Apply .gitignore to committed files
```

Comment:
I would call this "Untrack already committed files", for me it's clearer.

Reply:
I understand your point, we can talk about it on next meeting.